source/server/http/admin.cc

```diff
@@ -360,12 +379,19 @@ void AdminImpl::writeClustersAsJson(Buffer::Instance& response) {
         HEALTH_FLAG_ENUM_VALUES(SET_HEALTH_FLAG)
 #undef SET_HEALTH_FLAG
 
-        double success_rate = host->outlierDetector().successRate();
+        double success_rate = host->outlierDetector().successRate(
```

Comment:
nit: `const double`

Reply:
Done.

Reply:
Actually, success_rate is used later on in write operation. Must stay as is.